.vib/grafana-tempo-vulture/goss/grafana-tempo-vulture.yaml

```diff
@@ -0,0 +1,6 @@
+file:
+  ## tempo-vulture does not return the version so we check the license
+  /opt/bitnami/grafana-tempo-vulture/licenses/agpl-source-links.txt:
```

Comment:
This file is internally generated by us using the same metadata used to set `$APP_VERSION`, that is, they will always match. If we can't obtain the asset version directly from the asset binaries or file, we should avoid the check.